src/components/05_pages/Permissions/Permissions.js

```diff
@@ -108,7 +100,7 @@ const Permissions = class Permissions extends Component {
                 ) : (
                   <input
                     type="checkbox"
-                    onChange={e =>
+                    onChange={() =>
```

Comment:
I'm wondering: Is there a way to avoid creating new functions for each rendered checkbox?

Reply:
@dawehner Let me investigate that.